### DISCUSSION TITLE
Number Icon in Notification Area not displaying when screen locked

### DISCUSSION THREAD

**Discussion by michaeld427** [NightscoutFoundation/xDrip#2737] [General]
This started last night after installing the March security update on my Pixel 7 Pro.  The Xdrip app "Lock screen" setting (Android setting) is set to display all notifications.

**Comment by Navid200** [comment]
Number icon in notification area shows in the notification area.
Why would it show with the screen locked?

**Reply by michaeld427** [reply]
Why wouldn't it?  That is how it has always worked.  You don't need to fully unlock your phone to see the persistent notification.

**Reply by Navid200** [reply]
I'm not sure we are talking about the same thing.
If you are talking about lock screen, you can control that here: https://navid200.github.io/xDrip/docs/Lock-screen.html

If you want your phone notifications to be visible on lock screen, that's a phone setting.  Different phones have different behaviors.  Some phones have that feature.  Some don't.
If Google has decided to remove that from Pixel 7, there isn't much we can do in xDrip to change that behavior for Pixel 7.

**Reply by michaeld427** [reply]
No, I'm not talking about the "Number Wall on Lock Screen" option.  I do not use that because of a different existing issue (https://github.com/NightscoutFoundation/xDrip/issues/1891).

I am talking about the "Number Icon in Notification Area" Display setting, which displays your BG number as a notification.  This would also show on the lock screen previously and now it no longer does.

**Reply by Navid200** [reply]
I understand you say this worked before and is not working now.

If you can try different versions of xDrip and see that it has stopped working after a particular xDrip release, and it works again if you switch to an older xDrip, it will be a good sign that we changed something in xDrip causing this change of behavior and we can look into fixing it.

But, you said this happened after an Android patch.
That means that Google made a change that has caused this change in behavior.

What shows on your screen when the phone is locked is controlled by your phone manufacturer and Android.  xDrip does not have much to do with it.

**Comment by psjostrom** [comment]
I think you are trying to describe the same issue that I am having. But if we are having the same problem, I understand that the developers are having a hard time understanding the problem from this bug description.

I started a discussion describing my issue, please let me know if you have the same issue and we can continue the discussion there.

https://github.com/NightscoutFoundation/xDrip/discussions/2738

**Reply by michaeld427** [reply]
That wasn't what I was referring to, but now I'm thinking I was misremembering where I was see the BG on the lock screen.  I knew 100% that I didn't have to unlock my phone to see my kids BG.

But now that I got this graph back, maybe that it always where I got it from and it never displayed in the top notification area like I was thinking.
